Assets/Scripts/Tank/TankView.cs

```diff
@@ -4,8 +4,21 @@
 
 public class TankView : MonoBehaviour
 {
```

Comment:
Tank view is a class  that should be on prefab here and its work is to render object as well as using Update as well as fixed update and pass that data to controller.